tests/ClientTest.php

```diff
@@ -139,4 +140,83 @@ public function testGetTopLevelFolders() {
     $this->assertTrue(is_array($folder));
     $this->assertInstanceOf('cweagans\webdam\Entity\Folder', $folder[0]);
   }
+
+  /**
+   * Test getAsset.
+   */
+  public function testGetAsset() {
+    $mock = new MockHandler([
+      new Response(200, [], '{"access_token":"ACCESS_TOKEN", "expires_in":3600, "token_type":"bearer", "refresh_token":"REFRESH_TOKEN"}'),
+      new Response(200, [], file_get_contents(__DIR__ . '/json/asset.json')),
+    ]);
+    $handler = HandlerStack::create($mock);
+    $guzzleClient = new GClient(['handler' => $handler]);
+
+    $client = new Client($guzzleClient, '', '', '', '');
+
+    $asset = $client->getAsset(12345);
+    self::assertTrue(is_object($asset));
+    self::assertInstanceOf('cweagans\webdam\Entity\Asset', $asset);
```

Comment:
`$this->` instead of `self::` throughout. There is an instance of the current class, so calling these assertions statically is not needed.